LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 Anurag Singh
```

Comment:
Licensee name must be repo owner name and not yours. Wait till I give a green signal on this one!